manifest.json

```diff
@@ -147,6 +147,11 @@
       "description": "Do everything except actually executing the command line",
       "type": "boolean"
     },
+    "gear-keep-output": {
+      "default": false,
+      "description": "Don't delete output.  Output is always zipped into a single file for easy download.  Choose this option to prevent output deletion after zipping.",
```

Comment:
Does this mean that the output directory would be present, in addition to the zip file? 
What is the use-case for this?

Reply:
Yes, exactly.  Use-case?  Good question.  I added it to see what would happen and because it took 2 seconds (except for not being able to upload the gear at first).  I guess it might be useful if somebody wants to check an html file quickly on the platform without downloading a whole .zip file.
Should I take it out because "less is more"?